src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -98,13 +119,55 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 conn.disconnect();
             }
         }
+        
+        String elections = strBuf.toString(); 
+
+        JSONObject obj = new JSONObject(elections);
+        JSONArray electionData = obj.getJSONArray("elections");
+        int numElections = electionData.length();
+
+        for (int i = 0; i < numElections; ++i) {
+            JSONObject currentElection = electionData.getJSONObject(i);
+            Entity electionEntity = new Entity("Election");
+
+            /* The "id" of an Election Entity is stored as a property instead of
```

Comment:
nit: Can you follow one of these conventions for block comments?

https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Reply:
resolved in PR for "Comment Formatting".